packages/java/pom.xml

```diff
@@ -4,11 +4,40 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>org.aneo</groupId>
+  <groupId>fr.aneo</groupId>
```

Comment:
perhaps `eu.aneo` should be better. aneo.fr is only a redirection I think.

Reply:
the namespace has been created with the fr.aneo after discussing with Maxime